backend/src/services/memory/memory.service.ts

```
@@ -0,0 +1,272 @@
+
+  // ---- helpers ------------------------------------------------------------
+
+  private async embed(text: string): Promise<number[]> {
+    const res = await this.embeddingService.createEmbeddings({
+      model: EMBED_MODEL,
+      input: text,
+      dimensions: EMBED_DIMENSIONS,
+    });
+    const vec = res.data[0]?.embedding;
+    if (!Array.isArray(vec)) {
+      throw new AppError('Embedding provider returned no vector', 502, ERROR_CODES.AI_UPSTREAM_UNAVAILABLE);
```

ESLint

[error] Replace `'Embedding·provider·returned·no·vector',·502,·ERROR_CODES.AI_UPSTREAM_UNAVAILABLE` with `⏎········'Embedding·provider·returned·no·vector',⏎········502,⏎········ERROR_CODES.AI_UPSTREAM_UNAVAILABLE⏎······`

(prettier/prettier)